source/entities/DoorEntity.cpp

```diff
@@ -17,9 +17,10 @@
 
```

Comment:
Before we had the game entity listener, I couldn't bind trap entities with traps because one might be deleted before the other one. For this reason, I had used a string to check if the trap was still alive before using it. Now, that we have the listener, we can use the trap directly without fear ^^